index.js

```diff
@@ -4,6 +4,9 @@
 
 const path = require('path');
 const fs = require('fs');
+const { spawn } = require('child_process');
```

Comment:
Can / should we use `execa` instead of `child_process` directly? I _generally_ prefer `execa` (it has covered over many issues/downfalls of `child_process` for me in the past), but if you don't think it provides much benefit here I'm happy to go with your preference.

Whoops, just saw your comment RE: this in the PR description. Totally up to you though...

Reply:
Decided to use it as the code was shorter, and we already had it in the tests. May as well be consistent :)